### PR TITLE
must-gather: Fix path for ceph volume logs.

### LINKED ISSUE #828: must-gather: need improvement in collecting ceph-volume logs
The path needs be modified and whole code should have more verbose 

`# Collecting output of ceph volume commands
    for ((i = 0; i < ${#ceph_volume_commands[@]}; i++)); do
        printf "collecting command output for: %s\n"  "${ceph_volume_commands[$i]}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
        for osdPod in $(oc get pods -n "${ns}" -l app=rook-ceph-osd --no-headers | awk '{print $1}'); do
            pod_status=$(oc get po "${osdPod}" -n "${ns}" -o jsonpath='{.status.phase}')
            if [ "${pod_status}" != "Running" ]; then
                continue
            fi
            COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/${ceph_volume_commands[$i]// /_}
            { timeout 120 oc -n "${ns}" exec "${osdPod}" -- bash -c "${ceph_volume_commands[$i]}" >> "${COMMAND_OUTPUT_FILE}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
        done
    done`

### PR DESCRIPTION
**Pull request by rajatsing**
Fixed the path for ceph volume logs in the collection script.

Fixes #828

### CODE REVIEW

**Review by crombus — CHANGES_REQUESTED** [review]
Thanks for the PR. code changes looks good. just for the commit add `must-gather:` instead of `ceph:`

**Comment by rajatsing** [comment]
sure, thanks.

**Comment by rajatsing** [comment]
Done!. I will also attach the volume logs for a better conformation.

**Comment by crombus** [comment]
one doubt I see two ceph-volume.log files
[1]/host/var/lib/rook/openshift-storage/log/example-deviceset-0-data-0-rb9qz/ceph-volume.log 
[2]/host/var/lib/rook/openshift-storage/log/ceph-volume.log  

which one to take or both are same.

and name ocs-deviceset will it same for all the clusters.

**Comment by rajatsing** [comment]
Volume logs from one of the OSDs.
Screenshot 2020-11-24 at 5 43 50 PM

**Comment by rajatsing** [comment]
> one doubt I see two ceph-volume.log files
> [1]/host/var/lib/rook/openshift-storage/log/example-deviceset-0-data-0-rb9qz/ceph-volume.log
> [2]/host/var/lib/rook/openshift-storage/log/ceph-volume.log
> 
> which one to take or both are same.
Both are the same files. 
> and name ocs-deviceset will it same for all the clusters.
Yes,it's the same `ocs-deviceset-` followed by `0/1/2`.

**Comment by rajatsing** [comment]
@obnoxxx can you also please review this?.
This PR is based on #921 
Thanks.

File: must-gather/collection-scripts/gather_ceph_resources [hunk 1]
```diff
@@ -195,21 +195,22 @@ for ns in $namespaces; do
...
+    }
 
     for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
         { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
     done
 
     oc delete -f pod_helper.yaml
 
+    # creating a counter variable for collecting PID in array
+    idx=0
```
Comment (crombus) [inline must-gather/collection-scripts/gather_ceph_resources:210]: I think no need for this change

File: must-gather/collection-scripts/gather_ceph_resources [hunk 2]
```diff
@@ -221,17 +222,20 @@ for ns in $namespaces; do
...
         if $debug_pod_ready; then
+            echo "debug pod is ready"
+            volume_collection
             oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/crash/ "${CRASH_OUTPUT_DIR}"
         fi
     }
 
-    # creating a counter variable for collecting PID in array
-    idx=0
+
```
Comment (crombus) [inline must-gather/collection-scripts/gather_ceph_resources:231]: extra line

File: must-gather/collection-scripts/gather_ceph_resources [hunk 3]
```diff
@@ -195,21 +195,22 @@ for ns in $namespaces; do
...
         { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
     done
 
     oc delete -f pod_helper.yaml
 
+    # creating a counter variable for collecting PID in array
+    idx=0
     crash_collection(){
-        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 8m" &
```
Comment (crombus) [inline must-gather/collection-scripts/gather_ceph_resources:212]: any specific reason for increase in sleep.

**Review by crombus — CHANGES_REQUESTED** [review]
as discussed in chat the info vol logs is being collected so it looks good. just few changes required.

File: must-gather/collection-scripts/gather_ceph_resources (hunk 1 above)
Comment (rajatsing) [reply must-gather/collection-scripts/gather_ceph_resources]: sure,fxed

File: must-gather/collection-scripts/gather_ceph_resources (hunk 2 above)
Comment (rajatsing) [reply must-gather/collection-scripts/gather_ceph_resources]: sure,fxed

File: must-gather/collection-scripts/gather_ceph_resources (hunk 3 above)
Comment (rajatsing) [reply must-gather/collection-scripts/gather_ceph_resources]: sure,fxed

**Review by crombus — APPROVED** [review]
perfect!!.  Thanks for the PR.
/lgtm

**Pull request merged**